drivers/serial/uart_nrfx_uart.c

```diff
@@ -414,14 +492,8 @@ static int uart_nrfx_init(struct device *dev)
 	nrf_uart_txrx_pins_set(uart0_addr,
 			       DT_NORDIC_NRF_UART_UART_0_TX_PIN,
 			       DT_NORDIC_NRF_UART_UART_0_RX_PIN);
-
-#ifdef CONFIG_UART_0_NRF_FLOW_CONTROL
-#ifndef DT_NORDIC_NRF_UART_UART_0_RTS_PIN
-#error Flow control for UART0 is enabled, but RTS pin is not defined.
-#endif
-#ifndef DT_NORDIC_NRF_UART_UART_0_CTS_PIN
-#error Flow control for UART0 is enabled, but CTS pin is not defined.
-#endif
+#if	defined(DT_NORDIC_NRF_UART_UART_0_RTS_PIN) && \
```

Comment:
Please use `IS_ENABLED` macro.

Reply:
Functions called in lines below use value of this define, I can use #if with IS_ENABLED but I don't think it will improve anything. Using if() is not possible here